src/elements/core/styles/table.scss

```
@@ -0,0 +1,46 @@
+@use './mixins/table' as t;
```

maybe to be consistent use
```suggestion
@use './mixins/table';
```
and then table.*